### ISSUE TITLE
Strip out h1 from wysiwyg area

### ISSUE THREAD

**Issue by reubenmoes** [domain7/wp-plugin_boilerplate#4]
Just allow h2-h5 in wysiwyg would be cool

**Comment by nathansh** [comment]
Yes! I thought about this too, but I didn't capture that anywhere. We should definitely do this. Would this be better for the plugin, or maybe even a standalone plugin? Thinking data creation and storage vs data presentation.
